uniffi_core/src/ffi/foreigncallbacks.rs

```diff
@@ -195,9 +197,160 @@ impl ForeignCallbackInternals {
         };
     }
 
-    pub fn get_callback(&self) -> Option<ForeignCallback> {
+    fn call_callback(
+        &self,
+        handle: u64,
+        method: u32,
+        args: RustBuffer,
+        ret_rbuf: &mut RustBuffer,
+    ) -> c_int {
         let ptr_value = self.callback_ptr.load(Ordering::SeqCst);
-        unsafe { std::mem::transmute::<usize, Option<ForeignCallback>>(ptr_value) }
+        unsafe {
```

Comment:
We should have a `// SAFETY:` comment here documenting why what we do is in fact safe
(We previously got `ptr_value` from an `Option<ForeignCallback>` and stored that)

This was missing in the old code as well.